code/modules/asset_cache/assets/character_profile.dm

```diff
@@ -0,0 +1,14 @@
+/datum/asset/simple/character_profile
+	var/mob/living/carbon/human/host
```

Comment:
aren't assets cached forever once added
this might be an issue if you use a hard reference here given humans can be deleted